drivers/volume/portworx/portworx.go

```diff
@@ -3753,6 +3753,13 @@ func (d *portworx) SetClusterOpts(n node.Node, clusterOpts map[string]string) er
 	}
 
 	clusteropts = strings.TrimSuffix(clusteropts, " ")
+	//Create context with admin token if PX security is enabled.
+	if len(d.token) > 0 {
+		_, err := d.nodeDriver.RunCommand(n, fmt.Sprintf("%s context create admin --token=%s", d.getPxctlPath(n), d.token), opts)
```

Comment:
There is already a function `GetPxctlCmdOutputConnectionOpts()` for this https://github.com/portworx/torpedo/blob/b8439551d85a021544967bd0bb238558c56016d3/drivers/volume/portworx/portworx.go#L3995, it does create auth context, run cmd and deletes it afterwards, take a look and see if you can just use already existing func for this, as it takes care of multiple things there already?

Reply:
Thanks @nikolaypopov for the suggestion. I addressed review comments.